src/Controls/src/Core/VisualElement/VisualElement.cs

```diff
@@ -1174,7 +1192,11 @@ SizeRequest GetSizeRequest(double widthConstraint, double heightConstraint)
 		/// <param name="flags">A value that controls whether margins are included in the returned size.</param>
 		/// <returns>The minimum size that an element needs in order to be displayed on the device.</returns>
 		/// <remarks>If the minimum size that the element needs in order to be displayed on the device is larger than can be accommodated by <paramref name="widthConstraint" /> and <paramref name="heightConstraint" />, the return value may represent a rectangle that is larger in either one or both of those parameters.</remarks>
+#pragma warning disable RS0016 // Add public types and members to the declared API
+
+		[Obsolete("Use Measure with no flags.")]
```

Comment:
Is there a `Measure()` or did you mean `MeasureView`?

Reply:
I wonder if we can do something different:

```cs
// add a nice new overload
public Size Measure(double widthConstraint, double heightConstraint);

// obsolete the old one and remove the flags default parameter
[Obsolete]
public virtual SizeRequest Measure(double widthConstraint, double heightConstraint, MeasureFlags flags)
```

https://sharplab.io/#v2:C4LgTgrgdgNAJiA1AHwAICYAMBYAUBgRjz1QIE4AKAIkB4NwEH2qACRRqAUwHdGAVNgZ2AUAlADoAsmwCGfCGDYUCMRuiFCA3MXzoe/YHgDejI4zzHGqAMyMAbgEswwCJIA25gpkYTps+XAD2EABGzmyMHEr+QSGMABZKtlDAjABmjAC8jJhC6QB8jFRUGrimxpY29o4ujAlJXjJyFJHBoeGMTdEx2Wl5mEUAvkA

This works only because we have implicit operators to and from Size:

https://sharplab.io/#v2:C4LgTgrgdgNAJiA1AHwAICYAMBYAUBgRjzwGUBLALwFMAlKgRwioGdgACdgXjaioHc2AFRbAAFAEoAdAFkqAQ2YQwVUQRht04gNzF8BAJyiARIB4NwCD7Rtog7bdGISLwBvNq7Z43bVAGY2ANzIwYAg5ABs2cmo6RhE2WQUlFTgAewgAI1CqNj51FPTMtgALdTIodgAzNm5McSqAPh5+UV4+CXUWiVtcDzcfCMos+MVlUTyMrJy2MYLC2s4Gjq6AXztfAAc5ILIwtlZIAGN2SKpnPABIPtQCADY2ZTkUqFCAT37qNgAtKjBknW7cC6+Y6jVLjXYDKpsGrnJznM4AdTIcGAhUhzAGfzOZwAElQyABzQpccHUTErf6At5JUEFPhIlG5GlZQp4wnAcQwuGI5Go7h0nmYnGsomQlkEolk3SUtLJZLhACSzC+P3qbG5KKq1TYADJtWxceKuNU/uc+tMsurUS58VRgFpdrb7eTTb5zfrhexrY6HXa2M6AZcblS2Mk1t85MBkmArKJjrs1MHmJpOQCLgB2RoCYHMAiSS1WXboPP04rxyQGtkFpPlj1dM7+ylXW5x0PhyPRgC0sYhOfUcaTHIBsNTqAzLSpohzxZ5bA7henDLLFZFc+ry/ZkopgebENbYAjUbYACpu+9e1Mmf4wkxB2dh1jR5mJ1P80er6EmOop+vj++b5uXS8IMWzDfd2zYAB6U8snPN0/GvKhb3vdMn2zXN8wgv8qC/XMf0w+CP0QgCA18Js2BlOUQ1Ag9o04ThoPjPse2TIc4UfF8S01ed811JcPS4tcPWIxsgwo8I9xotgAEJ6P7BN+xYu82IzDiZxk7jOOQZA+MNaTuEEw1hL6MS2AAUUYMJmAY5IUW+JDlLVEtJHMkJQismyWTABdZh1PV12ciy3NEDzvhrQ06wbPpkj8b4wCRLITJcyzgrSAArKhDhDNL7NTMhKhS1K2DIZgeAgUJQlvB8M3KSyqEFR9kjSoqSrjXikqC6DxEa1KIqlKKYrAOK4CyUp2AAcVtbEFEKABhZJhokFMsWgfYWX2ABrKg4DhZCUNES1JAm4ApuYWb5pUWo328fQ01qAA9d1DUOybprmha63rc5/X9CDfrYAAeUp9g/Ya2HKMhMk4IxJEkCCYYglJ9mYCDpDIfYfmYZJymAGQ5AgMgIOOaImFYSQAA8AFtQksDYUShwRnjDABtVBUAAMTK0IADk5ApqhOAAclR9Hkkx7HcfxyQiYYEngAFgBdCCABFkiRiCj0sCC6kAvYIEy6WYlYNgQDYeUkuAORxn+g3Ze11jU1+zDAagYGIFB8HIeh2H4cR5HhYxrGcekPGCZtkRyapmmI0KKHftkCm0m+FnUHjxOwB5vnBeJkQFeV1XkY1yC7YfIEIWzo2vV9ZhvQbM5HYBoGQayD3+a9uHYd9lG0YD8Xg/xwmBnLnHKepthaZjow46oBOk9Z1Pvgz/mhdKMgKYgCnc5VtXC61tjS/eVGoFX9e2Er+1q99Wv6+d133Yh1v4Z9/Ou5FsWg5Dgeohl8OR6junJ5RtPNOyd57p15kvSQhwowKyZugRWW8C6a2LpSMOrAGLKENsAJi7wKYrzXhTSqu0h6QgwbLQUZxD7HwppCXBR98GCivn9G+Tcwb3yho/Duz9/ai0DhLUOg9v6k1/mPaOsdAEzzACAoBC9wGC0ge2GBBB4HPx3sgvoqCxBxlISIQhcIzjEO4No1g5DKH4JIYIu0cJGFO0bm7ZubC25PzVtwt+fDP60AsRHUe48xGgKkRIxegtBDJBIMAOKUB8SbxUUgveIYBpDSyFcTAQgQlhNKPiRaqZdqPiSZINmUYKYRmME4JwQ9OBOEwEsOIeD17lIIEsBpRh1BD3UKY9eH1a7GVlOEdqVkNEhlsmAOYDQh4BVcu5QZkgh61F4m0imYzkohS8nMrosSyKrzWKENGZB2ASXAhohi6JqDDNQgIzBk4BirJHKRIMGytn7B2VRNsh5gT9KOYhVU7ypkWJNNcuJsV4rkW6WZQK7k0oZWAAAfiyj1VU3VmpUmIUs3yILxnBUGVcku/zBqAtGmwI6J0zrvVVKMglr1zoSDYA9OZz1jrkver8rFZETJ7MPHRaCxDMjY2wR4zBdxwqqi5TjXpog4pskxSJW4LLqLgRkhyixbAhU8uIWKokJypKiCFVxVVG4vp4CAA